pkg/infra/pulumi_aws/iac/elasticache.ts

```diff
@@ -46,7 +47,10 @@ export const setupElasticacheCluster = (
         retentionInDays: 0,
     })
 
-    const clusterName = sanitizeClusterName(appName, dbName)
+    // TODO: look into removing sanitizeClusterName when making other breaking changes to resource names
+    const clusterName = sanitized(
```

Comment:
same here, need to remove clustername as the actual name for this and memdb but lets maybe chat about that tomorrow